melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/identifisering/PersonIdentifiseringService.java

```diff
@@ -35,24 +41,41 @@ public PersonIdentifiseringService(
         this.personSokMetrikker = personSokMetrikker;
     }
 
-    public Optional<String> identifiserPerson(SedHendelse sedHendelse, SED sed) {
-        Optional<FagsakRinasakKobling> eksisterendeSak = saksrelasjonService.finnVedRinaSaksnummer(sedHendelse.getRinaSakId());
+    public Optional<String> identifiserPerson(String rinaSaksnumer, SED sed) {
+        Optional<FagsakRinasakKobling> eksisterendeSak = saksrelasjonService.finnVedRinaSaksnummer(rinaSaksnumer);
 
         if (eksisterendeSak.isPresent()) {
             String aktoerID = sakService.hentsak(eksisterendeSak.get().getGsakSaksnummer()).getAktoerId();
             return Optional.of(personFasade.hentNorskIdent(aktoerID));
         }
 
-        Optional<String> ident = Optional.ofNullable(sedHendelse.getNavBruker());
-        if (ident.isPresent()) {
-            PersonSokResultat resultat = personSok.vurderPerson(ident.get(), sed);
+        Optional<Person> personFraSed = sed.finnPerson();
+        if (personFraSed.isEmpty()) {
+            personSokMetrikker.counter(SoekBegrunnelse.INGEN_PERSON_I_SED);
+            return Optional.empty();
+        }
+
+        return vurderPersonFraSed(personFraSed.get());
+    }
+
+    private Optional<String> vurderPersonFraSed(Person person) {
```

Comment:
`vurderPersonFraSed` virker litt for abstrakt for meg. Jeg tenker om noe i retning av `vurderEllerSøkPersonForIdentifisering` kunne være tydeligere.